addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -1455,18 +1454,7 @@ test("receive new message plays sound", async () => {
     await waitForSteps(["sound:new-message"]);
 });
 
-test("message sound on receiving new message (push notif enabled)", async () => {
-    // Simulate push notification allowed
-    patchWithCleanup(window.navigator, {
-        serviceWorker: Object.assign(new EventBus(), {
-            register: () => Promise.resolve(),
-            getRegistration: async () => ({
-                get pushManager() {
-                    return Promise.resolve({ getSubscription: async () => ({}) });
-                },
-            }),
-        }),
-    });
```

Comment:
Simulated push notification enabled is no longer enough: needs to make use of service worker code but this is not testable. "Downgrade" of test is to at least keep on/off message sound settings which is the current main value of test. Push notif test would be nice at some point in master by making service worker code testable.